Resources/Prototypes/_StarLight/Entities/Mobs/Species/thaven.yml

```
@@ -62,9 +62,9 @@
     proto: guardian
   - type: Vocal
     sounds:
-      Male: UnisexThaven
-      Female: UnisexThaven
-      Unsexed: UnisexThaven
+      Male: MaleThaven
+      Female: FemaleThaven
+      Unsexed: MaleThaven
```

```suggestion
      Unsexed: UnisexThaven
```
add sounds to unisex

---

Every species that has different sounds for male and female uses the male sounds for unsexed, so there's no reason to add sounds to unisex. The only species that don't do this are the ones that are more alien, and they usually just have unisex sounds for everything (such as moth, reptilian, arachnid, etc.).